tests/countries/test_india.py

```diff
@@ -34,124 +34,428 @@ def test_subdiv_deprecation(self):
 
     def test_2018(self):
         self.assertHoliday(
-            "2018-01-14",
             "2018-01-26",
             "2018-03-02",
             "2018-03-25",
+            "2018-03-29",
             "2018-03-30",
             "2018-04-01",
             "2018-05-01",
-            "2018-05-20",
-            "2018-06-15",
             "2018-06-15",
             "2018-08-15",
             "2018-08-21",
-            "2018-08-22",
+            "2018-08-26",
+            "2018-09-03",
+            "2018-09-13",
             "2018-09-20",
             "2018-10-02",
-            "2018-10-02",
+            "2018-10-10",
+            "2018-10-18",
+            "2018-10-19",
             "2018-11-07",
+            "2018-11-08",
+            "2018-11-14",
             "2018-11-20",
+            "2018-11-23",
             "2018-12-25",
         )
 
         subdiv_holidays_mapping = {
```

Comment:
Some states contain unnecessary dates (in example, 2018-08-15 - nation-wide holiday), some states don't contain all dates of local holidays; states without local holidays are redundant here. Also it's better to bring formatting to a single style, one date per line.